Frameworks/CoreGraphics/CGContextCairo.mm

```diff
@@ -540,8 +544,8 @@
 }
 
 void CGContextCairo::CGContextSetTextPosition(float x, float y) {
-    curState->curTextPosition.x = x;
-    curState->curTextPosition.y = y;
+    curState->curTextMatrix.tx = x;
```

Comment:
(this or GetTextPosition) did we say in an earlier meeting that the actual position is affected by the current text transform, or am I remembering wrong?